### PR TITLE
Update readme with NPM publish instructions

### PR DESCRIPTION
**Pull request by rszaloki**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,6 +99,12 @@ We are seeking for great minds to extend our core team. Contribution in any area
 
 **[Development environment setup](docs/developmentEnvironment.md)**
 
+## Publish NPM package
+
+When the abiniser directory changed:
+- bump the version property in the package.json
+- `npm publish --access public`
```
Comment (szerintedmi) [inline README.md]: why do we need `--access public` ? isnt that the default unless set differently in package.json?

**Comment by szerintedmi** [comment]
also its not clear to me which files  are actually published  ? where is it set?

File: README.md (hunk 1 above)
Comment (rszaloki) [reply README.md]: The default for scoped packages, that they are published as private package. You need that --access public to publish as a public package (at least the documentation said that :)

**Comment by rszaloki** [comment]
The files which gets published in the package is set in the package.json's files property (now: ["abiniser"])

**Pull request merged**